### PR TITLE
Test/with sub dir

### PR DESCRIPTION
**Pull request by wata727**

### CODE REVIEW

File: rails_app/Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    mime-types (2.99)
+    mini_portile2 (2.0.0)
+    minitest (5.8.4)
+    multi_json (1.11.2)
+    nokogiri (1.6.7.2)
+      mini_portile2 (~> 2.0.0.rc2)
+    rack (1.6.4)
+    rack-test (0.6.3)
+      rack (>= 1.0)
+    rails (4.2.5.1)
```
Comment (sumyapp) [inline rails_app/Gemfile.lock:79]: **[brakeman]**
- Rails 4.2.5.1 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 4.2.7.1 [:ref](https://groups.google.com/d/msg/ruby-security-ann/8B2iV2tPRSE/JkjCJkSoCgAJ)
- Rails 4.2.5.1 contains a SQL injection vulnerability (CVE-2016-6317). Upgrade to 4.2.7.1 [:ref](https://groups.google.com/d/msg/ruby-security-ann/WccgKSKiPZA/9DrsDVSoCgAJ)

File: rails_app/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+
+  def dirty
+    render text: params.require(:dirty)
```
Comment (sumyapp) [inline rails_app/app/controllers/application_controller.rb:7]: **[brakeman]**
- Unescaped parameter value rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

File: rails_app/Gemfile.lock (hunk 1 above)
Comment (wata727) [inline rails_app/Gemfile.lock:79]: **[staging][brakeman]**
- Rails 4.2.5.1 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 4.2.7.1 [:ref](https://groups.google.com/d/msg/ruby-security-ann/8B2iV2tPRSE/JkjCJkSoCgAJ)
- Rails 4.2.5.1 contains a SQL injection vulnerability (CVE-2016-6317). Upgrade to 4.2.7.1 [:ref](https://groups.google.com/d/msg/ruby-security-ann/WccgKSKiPZA/9DrsDVSoCgAJ)

File: rails_app/app/controllers/application_controller.rb (hunk 2 above)
Comment (wata727) [inline rails_app/app/controllers/application_controller.rb:7]: **[staging][brakeman]**
- Unescaped parameter value rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

**Pull request closed without merging**